lib/util.c

```diff
@@ -1460,6 +1460,61 @@ grn_inspect_limited(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj)
   return buffer;
 }
 
+grn_obj *
+grn_inspect_key_with_table(grn_ctx *ctx, grn_obj *buffer, grn_obj *table, const void *key, unsigned int key_size)
```

Comment:
Do we need `_with_table` suffix?
When should we inspect key without table information? Key is related to table. So table information is important.

Reply:
I see, I've fixed